packages/dapp-toolkit/src/modules/wallet-request/crypto/blake2b.ts

```diff
@@ -0,0 +1,26 @@
+import type { Result } from 'neverthrow'
+import { err, ok } from 'neverthrow'
+import blake from 'blakejs'
+import { Buffer } from 'buffer'
+
+const bufferToArrayBuffer = (buffer: Buffer): ArrayBuffer => {
+  const arrayBuffer = new ArrayBuffer(buffer.length)
+  const view = new Uint8Array(arrayBuffer)
+  for (let i = 0; i < buffer.length; ++i) {
+    view[i] = buffer[i]
+  }
+  return arrayBuffer
+}
+
+const bufferToUnit8Array = (buffer: Buffer): Uint8Array =>
+  new Uint8Array(bufferToArrayBuffer(buffer))
+
+export const blake2b = (input: Buffer): Result<Buffer, Error> => {
```

Comment:
I'd appreciate adding unit 
```
describe('blake2b', () => {
  it('should hash a string', async () => {
    const result = await blake2b(Buffer.from('test'))

    if (result.isErr()) throw result.error

    expect(result.value.toString('hex')).toBe(
      '928b20366943e2afd11ebc0eae2e53a93bf177a4fcf35bcc64d503704e65e202',
    )
  })

  it('should hash base64 string synchronously', () => {
    const hash = blakeHashHexSync('74657374')
    expect(hash).toBe(
      '928b20366943e2afd11ebc0eae2e53a93bf177a4fcf35bcc64d503704e65e202',
    )
  })
})
```

I'd like to remove this function from CE and use this one. Please just one unit test to be on the safe side in the future. [CE implementation](https://github.com/radixdlt/connector-extension/blob/main/src/crypto/blake2b.ts)

Reply:
added